packages/x-translations/.versionrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Do we need this file? Been checking other packages and cannot find this file in them.